docs/deployment/connect-scm.md

```diff
@@ -33,38 +36,54 @@ The process to connect a source code manager depends on whether your SCM tool is
 
 If you opted to scan a GitHub or GitLab repository when you initially signed in, you may have already performed these steps and can skip to [Next steps](#next-steps).
 
+<Tabs
+    defaultValue="github-cloud"
+    values={[
+    {label: 'Azure DevOps Cloud', value: 'azure-devops-cloud'},
+    {label: 'Bitbucket Cloud', value: 'bitbucket-cloud'},
+    {label: 'GitHub Cloud', value: 'github-cloud'},
+    {label: 'GitLab Cloud', value: 'gitlab-cloud'},
+    ]}
+>
+
+<TabItem value='azure-devops-cloud'>
+
 ### Azure DevOps Cloud
 
-1. Sign in to Semgrep AppSec Platform.
-1. On the sidebar, click the account name to open the drop-down menu.
-1. Using the drop-down menu, select the account you want to make a connection for.
-1. Go to **Settings** > **Source Code Managers**, and click **Add Azure DevOps**.
-![Source code manager tab](/img/source-code-manager.png#md-width)
+1. Sign in to [<i class="fas fa-external-link fa-xs"></i> Semgrep AppSec Platform](https://semgrep.dev/login).
+1. Optional: If you have created more than one Semgrep account, select the account you want to make a connection for by clicking on the **Navigation bar > Your account name > The account you want to connect**.
+   <img src="/docs/img/more-accounts-dropdown.png" height="350px" />
```

Comment:
Not a big deal, but I think the screenshot should be on its own line (this is present for all tabs)

Reply:
Oh dear. It's probably formatted slightly differently for our desktops, I'll add a br.